packages/typespec-vscode/test/manual/generate-openapi3-from-typespec-test-plan.md

```diff
@@ -0,0 +1,102 @@
+# Generate OpenAPI 3.x from TypeSpec Test Scenario
```

Comment:
I think we can merge the client, server stub and openapi3 user scenario into one, they are basically the same except choosing different emitters